addons/mail/static/src/messaging/test_utils.js

```diff
@@ -213,12 +252,14 @@ function _useMessagingMenu(callbacks) {
 
 /**
  * @param {Object} [param0={}]
+ * @param {boolean} [env]
  * @param {boolean} [hasChatWindow]
+ * @param {boolean} [hasLegacyMail]
  * @param {boolean} [isDebug]
  * @returns {Object}
  */
-function getServices({ hasChatWindow, isDebug } = {}) {
-    return new MockMailService().getServices({ hasChatWindow, isDebug });
+function getMailServices({ env, hasChatWindow, hasLegacyMail, isDebug } = {}) {
```

Comment:
any particular reason to rename? Don't think it's worth

Reply:
One click "go to definition" instead of indirection nightmare.